spec/integration/pgpapi.spec.js

```diff
@@ -0,0 +1,31 @@
+/*globals describe, beforeEach, require, expect, it */
+/*jslint indent:2,white:true,sloppy:true,node:true*/
```

Comment:
Yeah had that in from copy-pasting from other files, removed from all the files and they still lint fine. Will add that plus a few other things in next commit.